project-base/app/web/imageResizer.php

```diff
@@ -27,7 +27,7 @@
     exit();
 }
 
-$allowedImageSizes = [30, 50, 200, 350, 500, 1000];
+$allowedImageSizes = [16, 24, 32, 48, 64, 96, 128, 256, 480, 768, 1024, 1440];
```

Comment:
slider banners use 1400px. Why 1440?
Also, product lists use 180px. These numbers feels really random

Reply:
These sizes are generally used without large spaces. 

The browser chooses how big image it will download, so even if we have a 180px image, there is a high probability that the browser will not download such an image. 

Now it's synchronized with storefront and storefront will generate srcSet with those sizes by default